bin/express-cli.js

```diff
@@ -442,6 +458,7 @@ function main () {
     if (program.hbs) program.view = 'hbs'
     if (program.hogan) program.view = 'hjs'
     if (program.pug) program.view = 'pug'
+    if (program.jsx) program.view = 'jsx'
```

Comment:
If you move the `if` statement up a couple lines it'll keep the list alphabetical.